### ISSUE TITLE
Better "no tables" view in database/tables

### ISSUE THREAD

**Issue by saltcod** [supabase/supabase#15425]
When you first create a project and go to the database tables page, it’s not clear to some customers what it’s meant for or how to use it. Easy to miss the + New table button in the top right, especially on large screens.

An interface like Functions has would be better here.

<img width="588" alt="CleanShot 2023-06-28 at 14 12 33@2x" src="https://github.com/supabase/supabase/assets/105593/ea7627fb-5246-44c7-98d9-387350d26446">

**Comment by takanome-dev** [comment]
Hey @saltcod, how about adding another button `new table` just underneath the placeholder?

Btw, I would like to take this :slightly_smiling_face:

**Comment by gregnr** [comment]
@takanome-dev that sounds great! Ideally let's try to keep a consist UI with the functions (ie. title, description of what the feature does, 'Create a new table' button)

**Comment by takanome-dev** [comment]
Cool, one more question @gregnr 
Should I remove the `NoSearchResult` component or simply change the content inside?

There is also a note on that component:
https://github.com/supabase/supabase/blob/3f68a2019c2b8eeb1dfc75e1a30c1a18c87f3a0a/studio/components/to-be-cleaned/NoSearchResults.tsx#L4

The `NewSearchResult` component in [components/ui](https://github.com/supabase/supabase/blob/master/studio/components/ui/NoSearchResults.tsx)mentionned does not match the UI with the functions

**Comment by takanome-dev** [comment]
Also I'm getting a `500` error when I start the dev server for the studio and navigate to the databases table page. Opened a draft pr to be able to see the changes. Any help is appreciated

**Comment by abdqudus** [comment]
Could I contribute to this project?

**Comment by saltcod** [comment]
@abdqudus looks like this one is already in hand, but feel free on any of the other "Good first issues"

**Comment by gregnr** [comment]
@takanome-dev Yes let's match what Functions does w.r.t. the component being used. Worth considering - we should really have 2 separate types of empty state: one where it's a brand new project and no tables have been created yet (what this issue is about), and a second when you do have tables, but after using the search filter no results are returned (basically the current message). Hope this makes sense - let me know if you have any questions.

**Comment by takanome-dev** [comment]
Thanks for the clarification, @gregnr.

Marking the pr as ready for review. I still cannot see the changes live (not on dev and even the deploy preview :thinking:), not sure if I'm doing something wrong.

Let me know if there are changes to make.

**Comment by Vedant767** [comment]
Hey, is this issue fixed? If not then I can handle this one.

**Comment by takanome-dev** [comment]
Hey @Vedant767, a pr has already been opened. Feel free to look on other issues :slightly_smiling_face:
